tests/swift/test_worker.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+pushd $(dirname "$0")
+DIR=$(pwd)
+RUNBOX="${DIR}/runbox"
+
+echo $RUNBOX
+# Create runbox
+mkdir -p $RUNBOX
+
+# Copy source to runbox
+cp -fv $DIR/script.swift $RUNBOX/script.swift
+cp -fv $DIR/run.stdin $RUNBOX/run.stdin
+
+# Test Compile
+docker run \
+    --cpus="1" \
+    --memory="100m" \
+    --ulimit nofile=64:64 \
+    --rm \
+    --read-only \
+    -v "$RUNBOX":/usr/src/runbox \
+    -v "$RUNBOX":/tmp \
+    -w /usr/src/runbox codingblocks/judge-worker-swift \
+    bash -c "/bin/compile.sh && /bin/run.sh"
+
+ls -lh ${RUNBOX}
+
+expected="Hello World"
+actual="$(cat ${RUNBOX}/run.stdout)"
+if [ "$expected" == "$actual" ] ;then
+:
+else
+echo "MISMATCH: Expected = $expected; Actual = $actual"
+exit 1
+fi
+
+# Delete runbox
+sudo rm -rf $RUNBOX
```

Comment:
Travis fails with 'No permission to rm runbox'. Hence sudo.

Reply:
@championswimmer Here is the error. Perhaps it has something to do with the permissions granted from where we are pulling our docker file? 

![screen shot 2018-07-11 at 7 57 39 pm](https://user-images.githubusercontent.com/29275810/42581076-93392776-8549-11e8-833b-c7ea4abaa695.png)

Reply:
This still persists. 
Local tests are running fine without sudo permission, however travis fails without it.
@championswimmer

Reply:
Local tests and travis passing